libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -3,6 +3,11 @@
 
 #include "config.h"
 
+/* Defind the maximum header size for http_parser to keep track of for us. */
+#define HTTP_MAX_HEADER_SIZE    HTTP_MAX_RESPONSE_HEADERS_SIZE_BYTES
```

Comment:
What is the purpose of duplicating the `HTTP_MAX_RESPONSE_HEADERS_SIZE_BYTES`? 
Also, this kind of configuration can be included in the `http_config.h` file for users to easily configure.

Reply:
I am trying to hide http-parser and add documentation for the setting in the context of our library.

Reply:
Updated in unseen code(next PR) to use http_parser_set_max_header_size.